src/tarkov-data-manager/index.mjs

```diff
@@ -1784,7 +1784,7 @@ app.post('/wipes', async (req, res) => {
         console.log(`creating wipe: ${req.body.start_date} ${req.body.version}`);
         const result = await query('INSERT INTO wipe (start_date, version) VALUES (?, ?)', [req.body.start_date, req.body.version]);
         let lastPriceId = 0;
-        const lastPrice = await query('SELECT id FROM price_data ORDER BY id DESC LIMIT 1');
+        const lastPrice = await query('SELECT id FROM price_data WHERE pve=0 ORDER BY id DESC LIMIT 1');
```

Comment:
That's a good point, but hopefully they will never add another game mode with a whole other flea market. It works as-is currently, so I'm not too inclined to change it until such time that it's necessary lol

Reply:
But when _such time that it's necessary_ comes we'll need to alter the column table name or add a new column instead of just keep using what we already have 🤷‍♂️ but I'm ok anyway 😉